app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  before_save {  email.downcase! }
+  validates :name, presence: true, length: { maximum: 50 }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
+  validates :email, presence: true, length: { maximum: 255 }, format: { with: VALID_EMAIL_REGEX }, uniqueness: true,uniqueness: { case_sensitive: false }
+  has_secure_password
+  validates :password, presence: true, length: { minimum: 6 }
+end
```

Comment:
I'll ask @oratilemlambo to add you to her repo so you can have a look at the comments I made there on [the same PR](https://github.com/oratilemlambo/sample_app/pull/4), that way I don't have to repeat myself :)

Reply:
Thanks Reviewed the changes on her pr swell